MediaAdminBundle/Resources/public/coffee/groupTree/FormMediaFolderCollectionView.coffee

```diff
@@ -0,0 +1,36 @@
+FormMediaFolderCollectionView = OrchestraView.extend(
+  events:
+    'change .folder-value-holder': 'changeInput'
+  initialize: (options) ->
+    @options = options
```

Comment:
can use indicate the differents options in comment, and use reduce option method